client/src/components/Chat.jsx

```diff
@@ -213,60 +213,56 @@ const Chat = () => {
     // Here whenever user will submit message it will be send to the server
     const handleSubmit = async (e) => {
         e.preventDefault();
-
+    
+        // Emit a typing event to indicate the user has stopped typing
         socket.emit(NEW_EVENT_TYPING, { chatId: app.currentChatId, isTyping: false });
-        const d = new Date();
-        let message = inputRef.current.value.trim();        // Trim the message to remove the extra spaces
-
-        if (!isQuoteReply) {
-            const cleanedText = message.replace(/>+/g, '');
-            message = cleanedText
-        }
-
-        if (message === '' || senderId === undefined || senderId === '123456') {
+    
+        const currentTime = new Date().getTime();
```

Comment:
Why did you change this?

Reply:
if (message === '' || senderId === undefined || senderId === '123456') {
    return;
}

This condition checked if the message was empty or if the senderId was undefined or equal to '123456'. it did not consider the case where the message might consist of only spaces or tabs, allowing empty messages with whitespace to be sent.

Reply:
That's not what I'm talking about, I'm talking about the current time